src/config/sns.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "SNS",
+  "displayName": "Simple Notification Service",
+  "rules": [
+    {
+      "aws": {
+        "logGroup": "sns/%{DATA:region_id}/%{DATA:account_id}/%{USERNAME:resource_id}%{GREEDYDATA:failure_suffix}"
```

Comment:
I think there is a test for grok matching on log group. Please add multiple testcases for this specific pattern so we can see more clearly how this matching with USERNAME works :) I am not sure how this works